modules/local/mt_deletion_scritp.nf

```diff
@@ -0,0 +1,43 @@
+process MT_DELETION {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::samtools=1.16.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.16.1--h6899075_1' :
+        'quay.io/biocontainers/samtools:1.16.1--h6899075_1' }"
+
+    input:
+    tuple val(meta), path(stats)
+
+    output:
+    path '*.txt'       , emit: mt_del_script
```

Comment:
Could you rename the outfile as the output isn't a script? We could call it just `mt_del` or `mt_del_result` or something else :)

Reply:
Also maybe add the meta to the output see Rams recent PR

Reply:
Done